frontend/src/bundles/talent-onboarding/components/profile-step/profile-step.tsx

```diff
@@ -214,6 +214,7 @@ const ProfileStep: React.FC = () => {
                     errors={errors}
                     name={'salaryExpectation'}
                     adornmentText="$"
+                    value={undefined}
```

Comment:
Why we need to add this line ? If value={undefined} just remove it from props ?

Reply:
We need it to remove '0'  as a predefined value for this input. Right now default value type is defined as number and we can`t set default value = empty string for salaryExpectation (or otherwise we would need to change dto type, joi schema and so on). By setting value={undefined} there won`t be any React errors with controlled/uncontrolled component, we will have empty input field with our required placeholder and form validation is still works as expected.

Reply:
How we can apply some value if you hardcode it?

Reply:
@nikita-remeslov  We already merged the same approach in mobile version. This input value undefined only on first render, but onChange it will be changed and correctly saved in redux/validated via react hook form
![Screenshot_2](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/67747035/da01d0d2-9e7e-46be-b7d2-e06890f4a78e)

Reply:
![Screenshot_1](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/67747035/f9a5bdc7-a40a-4a03-83fc-ba54b9eda2dd)